### ISSUE TITLE
Colorful tags not useful in suggested completions

### ISSUE THREAD

**Issue by wm75** [galaxyproject/galaxy#15726]
**Describe the bug**
When adding tags to histories in the User -> histories view, tags get suggested as you type, but with 23.0 those suggestions reuse the suggested tags' colors. The problem with that is that when using keyboard navigation, you cannot see which of the suggestions is currently selected.

**Galaxy Version and/or server at which you observed the bug**
Galaxy Version: 23.0rc1

**To Reproduce**
1. Go to 'User -> Histories'
2. Click on any history's tag icon and start typing (if you haven't tagged any histories previously repeat for a second history)
3. use tab to move across suggested completions
4. Hit enter to use the currently selected suggestion

**Expected behavior**
The current selection should be clearly highlighted

**Screenshots**
![image](https://user-images.githubusercontent.com/4742081/223421776-3185056e-2c93-4052-a4b6-7a2807983e13.png)

Not sure if this issue with tags exists anywhere else in the UI. Most other places I checked don't seem to be affected.

**Comment by hujambo-dunia** [comment]
Did my changes fix the other issue...?

**Comment by davelopez** [comment]
This is fixed with the new tag selector component.
